charts/generic-service/prison-postgres-restore.sh

```diff
@@ -9,15 +9,16 @@ psql_preprod() { psql -h "$DB_HOST_PREPROD" -U "$DB_USER_PREPROD" -d "$DB_NAME_P
 # grab last restore date from Prison API
 if ! DATABASE_RESTORE_INFO=$(check_http GET "$PRISON_API_BASE_URL/api/restore-info"); then
   echo -e "\nUnable to find any restore information."
-  if [[ -z "$FORCE_RUN" ]]; then
+  if [[ -z "${FORCE_RUN+x}" ]]; then
```

Comment:
FORCE_RUN might not be set, so have to default it to prevent bash complaining about variable not being defined